README.md

```diff
@@ -21,6 +21,8 @@ Then generate your new project:
 yo terra-module
 ```
 
+This will generate a project in the packages directory of a mono repo.  Make sure to type something other than the default when prompted for the project name.  After generating the project inside of the mono repo, you will need to add a link to your project in site/App.jsx, routes for your project in site/Index.jsx, and links to your test in site/TestLinks.jsx.
```

Comment:
@Matt-Butler, @bjankord and I had talked briefly about this.  We thought it was more important to get the generator out there and running for individual packages that to potentially add some complicated logic to modify existing files.  This is something that can be done later if we want.